pom.xml

```diff
@@ -39,15 +39,27 @@
     <properties>
         <project.scala.version>2.11.12</project.scala.version>
         <project.scala.binary.version>2.11</project.scala.binary.version>
-
-        <neo4j.version>3.5.12</neo4j.version>
-        <driver.version>1.7.5</driver.version>
-        <spark.version>2.4.0</spark.version>
+        <neo4j.version>4.0.0</neo4j.version>
+        <driver.version>4.0.0</driver.version>
+        <spark.version>2.4.1</spark.version>

```

Comment:
Spark 2.4.4 is the latest and greatest spark version out there. Might be good to use that one.

Reply:
Other than that LGTM